api/apps/articles/model/retrieve.coffee

```diff
@@ -33,6 +33,7 @@ moment = require 'moment'
       query.biography_for_artist_id = ObjectId input.biography_for_artist_id
     query.featured_artwork_ids = ObjectId input.artwork_id if input.artwork_id
     query.tags = { $in: input.tags } if input.tags
+    query.internal_tags = { $in: input.internal_tags } if input.internal_tags
 
```

Comment:
Noticed that vertical and subvertical are in the query schema, should they feature in here as well?